.github/renovate.json

```
@@ -51,7 +52,7 @@
     },
     {
       "matchDepTypes": ["devDependencies"],
-      "matchPackagePatterns": ["^jest-", "jest$", "^prettier", "prettier$"],
+      "matchPackagePatterns": ["^jest-", "jest$", "lint-staged", "markdownlint-cli", "^prettier", "prettier$"],
```

```suggestion
      "matchPackagePatterns": ["^jest-", "jest$", "lint-staged", "lockfile-lint", "markdownlint-cli", "^prettier", "prettier$"],
```

Suggested by @TimDiekmann in https://github.com/hashintel/hash/pull/3050#discussion_r1312959189